source/URL.ts

```diff
@@ -10,13 +10,18 @@ export interface URLData<E = unknown> {
     [key: string]: JSONValue | JSONValue[] | URLData | URLData[] | E;
 }
 
-export function parseURLData(raw = window.location.search): URLData {
+export function parseURLData(
+    raw = window.location.search,
+    toJSON = true
```

Comment:
```suggestion
    toBuildIn = true
```
JSON 是**语言无关的数据交互格式**，JS 里的 `toJSON` 都是在转换为 JSON 序列化后的数据，而非解析后的数据。